0/setup.md

```diff
@@ -35,7 +29,7 @@ The final tool we will be installing is the ink! command line utility which will
 You can install the utility using Cargo with:
 
 ```bash
-cargo install cargo-contract --vers 0.6.2 --force
+cargo install cargo-contract --vers 0.7.0 --force
```

Comment:
When will this be published?

Reply:
@ascjones will publish it today evening. Maybe as `rc1` though.

Reply:
https://github.com/paritytech/cargo-contract/releases/tag/v0.7.0

Reply:
https://crates.io/crates/cargo-contract/0.7.0